### DISCUSSION TITLE
Streammer Online - Radiocaster Cloudflare & Nginx

### DISCUSSION THREAD

**Discussion by kammateo** [AzuraCast/AzuraCast#5740] [Q&A]
Hello guys,

I have my own server and it works fine through HTTPS. I would like to use Radiocaster but it cannot connect to my server.
![image](https://user-images.githubusercontent.com/89619002/190864181-78f66d70-8213-412f-9de1-e458315d79ca.png)

 To avoid cloudflare, in Radiocaster I am using the IP address (see photo)
![image](https://user-images.githubusercontent.com/89619002/190864060-78e617dc-3553-4ab9-a3d7-8245b5fbd05e.png)

I don't know if the problem is because Ngnix blocks the port. 
![image](https://user-images.githubusercontent.com/89619002/190864340-c9c3d4ae-39d6-409e-bedb-743bb4025a21.png)

Do you know how I could fix it? Any hint you can give me would be very helpful.

I am using: 
 - Azuracast: AzuraCast • v0.17.3 Stable • Docker • PHP 8.1
 - Radiocaster 2.9.0.2
 - Cloudflare
 - Nginx

Cheers

**Comment by Mozitux** [comment]
If you have your server at your home, you must check that the ports concerned are open on your internet box.

**Reply by kammateo** [reply]
Thanks for the replay, in this case should be the port 8005?
Thank you for your help

**Reply by Mozitux** [reply]
Yes, if you use Icecast as broadcast software, you should open the port 8005 on your internet box.

**Reply by kammateo** [reply]
Hi @Mozitux I opened the port 8005 on my internet router but I still not having connection on the encoder. Do you have any idea what else it could be?

**Reply by Mozitux** [reply]
Are you communicating with your server via the IP address and not the domain name?

**Reply by kammateo** [reply]
Yes, I'm communicating with my Public IP 197.93.xx.xx  Address and the port 8005 is open on my router.
